VAMobile/e2e/tests/SignIn.e2e.ts

```diff
@@ -0,0 +1,45 @@
+import { CommonE2eIdConstants } from './utils'
+import { by, device, element, expect, waitFor } from 'detox'
+import { setTimeout } from 'timers/promises'
+
+export const SignE2eIdConstants = {
+  LOGIN_PAGE_ID: 'Login-page',
+  LOA_P1_TEXT: 'Before we give you access to your VA claim and health care information, we need to make sure you’re you. This helps us protect you from fraud and identity theft.',
+  LOA_P2_TEXT: 'If you haven’t yet verified your identity, we’ll help you complete the process when you sign in.',
+  LOA_GATE_EXPAND_MSG_TEXT: "Read more if you haven't yet verified",
+  LOA_GATE_READ_MORE_P1:
+    'We’ll verify your identity through a secure process from ID.me or Login.gov. This trusted partner provides the strongest identity verification system available.',
+  LOA_GATE_READ_MORE_P2: 'To complete the process on your smartphone, you’ll need these items:',
+  CONTINUE_SIGN_IN_BTN_ID: 'Continue to sign in',
+}
+
+describe('Sign In', () => {
+  it('should show sign in page content', async () => {
+    await waitFor(element(by.id(SignE2eIdConstants.LOGIN_PAGE_ID)))
+      .toExist()
+      .withTimeout(2000)
```

Comment:
This 2-second timeout can cause the test to fail if someone's machine is slow, or when running on CI. I found this failure happening locally for me some of the time when running these tests

Reply:
That's a good point. I also wonder if we should create constants for our timeouts based on how fast/slow we need them to be. I noticed we have a timeout of `10000` in a few tests, so it might be worth consolidating that

Reply:
Is that something that should changed in this PR or will another ticket be created for that change? @alexandec @theodur